app-modules/ai/src/Notifications/SendAssistantTranscriptNotification.php

```diff
@@ -98,6 +101,18 @@ public function toEmail(object $notifiable): MailMessage
         return $message;
     }
 
+    protected function afterSendHook(object $notifiable, OutboundDeliverable $deliverable): void
+    {
+        if (Feature::active('ai_utilization')) {
+            $this->thread->emailed_count = $this->thread->emailed_count + 1;
+            $this->thread->save();
+        }
+
+        if (Feature::active('ai_utilization') && Cache::has('special_actions_doughnut_chart')) {
+            Cache::forget('special_actions_doughnut_chart');
```

Comment:
You may want to make this more generic like `thread-emailed-count`

Reply:
As you added comment to remove cache from that chart, we didn't need this anymore.